research-2023/effort-responsibility-sharing/2023-03-08.003.md

```diff
@@ -0,0 +1,99 @@
+> This template will be used to guide the output of each spiral conducted within a research topic folder.
+
+# Initiate:
+
+## Define: Customer Responsibility Model
+
+## Problem
+
+As an OSCAL model developer, I need examples of the SSP model that contain content for export to a responsibility model based on #1336 and #1385. Responds to: https://github.com/usnistgov/OSCAL/issues/1467
```

Comment:
Should the sentence: "I need examples of the SSP model that contain content for export" read "I need examples of OSCAL SSP instances that contain content available for export"